baked_in.go

```diff
@@ -177,7 +178,8 @@ func parseOneOfParam2(s string) []string {
 	oneofValsCacheRWLock.RUnlock()
 	if !ok {
 		oneofValsCacheRWLock.Lock()
-		vals = strings.Fields(s)
+		re := regexp.MustCompile(`'[^']*'|\S+`)
```

Comment:
Can this be moved out out the hotpath? eg. a global variable 🙏

Reply:
Good idea, I will do that.